sklearn/discriminant_analysis.py

```diff
@@ -623,11 +623,15 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
         Discriminant Analysis
     """
 
-    def __init__(self, priors=None, reg_param=0., store_covariances=False,
-                 tol=1.0e-4):
+    def __init__(self, priors=None, reg_param=0.,store_covariance=False, tol=1.0e-4,store_covariances=None):
+        if store_covariances is not None:
+        	warnings.warn("'store_covariances' was renamed to 'store_covariance' in version 0.17 and "
+                      "will be removed in later versions.", DeprecationWarning)
+        	self.store_covariance = store_covariances
```

Comment:
You need to make the value of the *parameter* `store_covariance` equal to that of the `store_covariances` if it is passed. This means, you need to replace:
```
self.store_covariance = store_covariances
```
With:
```
store_covariance = store_covariances
```